### PR TITLE
Pred sound tweaks 

### PR DESCRIPTION
**Pull request by Joelampost**
## About The Pull Request
Adds more predator pain screams and tweaked the roar2 emote to sound closer to its intended sound.

## Why It's Good For The Game
Having only two pain screams for preds is terrible for your ears so I added more variety. 

## Changelog

:cl: Joe Lampost

soundadd: added three new pain screams for preds, tweaked roar2.
:cl:

### CODE REVIEW

File: code/game/sound.dm [hunk 1]
```diff
@@ -297,6 +297,8 @@
 			if("bone_break")
 				S = pick('sound/effects/bone_break1.ogg','sound/effects/bone_break2.ogg','sound/effects/bone_break3.ogg','sound/effects/bone_break4.ogg','sound/effects/bone_break5.ogg','sound/effects/bone_break6.ogg','sound/effects/bone_break7.ogg')
 			//misc mobs
+			if("pred_pain")
+				S = pick('sound/voice/pred_pain1.ogg','sound/voice/pred_pain2.ogg','sound/voice/pred_pain3.ogg','sound/voice/pred_pain4.ogg','sound/voice/pred_pain_rare1.ogg')
```
Comment (stanalbatross) [inline code/game/sound.dm:301]: you need to use weights in this pick() like the male_pain pick() to make the rare pain actually rare

**Review by stanalbatross — CHANGES_REQUESTED** [review]

**Review by stanalbatross — APPROVED** [review]
code approved, sound needs approval tho

**Review by stanalbatross — CHANGES_REQUESTED** [review]
do changes to predpain4 i told you about in the disc

**Comment by Joelampost** [comment]
I messed up the re-basing and as these changes are not urgent I decided to just close this PR.

**Pull request closed without merging**